micro-ui/web/micro-ui-internals/packages/modules/core/src/pages/employee/Login/login.js

```
@@ -71,9 +71,10 @@
 
     const requestData = {
       ...data,
+      ...defaultValues,
       userType: "EMPLOYEE",
     };
-    requestData.tenantId = data?.city?.code || Digit.ULBService.getStateId();
+    requestData.tenantId = requestData?.city?.code || Digit.ULBService.getStateId();
     delete requestData.city;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)